### DISCUSSION TITLE
Node down, can't recover funds, have seed phrase, SCB and wallet passphrase

### DISCUSSION THREAD

**Discussion by tioneb22** [lightningnetwork/lnd#7764] [Troubleshooting] [answered]
My LND node ran out of HDD space and I can't add more space as my ESX server is full.

I was able to mount my nodes HDD to another VM and I have full access to the lnd and bitcoin directories.

I have everything that is needed to recover my funds:
- 24 words
- Latest SCB backup
- Wallet passphrase

I tried the following to recover my funds but I keep getting the error "invalid passphrase"
- Tried entering SCB file and 24 words on a Blixt mobile wallet -> get invalid passphrase
- Tried setting up a fresh bitcoind and lnd on Debian 12 and using the "lncli create -multi_file=channel.backup" -> get invalid passphrase

Here are the detailed logs from my fresh lnd / bitcoind install:
- From LND: `[ERR] RPCS: [/lnrpc.WalletUnlocker/InitWallet]: invalid passphrase`
- From running lncli command: `[lncli] rpc error: code = Unknown desc = invalid passphrase`

What am I doing wrong? I'm pretty tech savvy but for some reason I can't figure this one out! Hopefully someone can help, I have 14m sats to recover!

Cheers in advance!

EDIT (lnd.conf and bitcoin.conf)

LND
```
[Application Options]
alias=YOUR_FANCY_ALIAS
debuglevel=info
maxpendingchannels=5
listen=localhost

# Password: automatically unlock wallet with the password in this file
# -- comment out to manually unlock wallet, and see RaspiBolt guide for more secure options
wallet-unlock-password-file=/data/lnd/password.txt
wallet-unlock-allow-create=true

# Automatically regenerate certificate when near expiration
tlsautorefresh=true
# Do not include the interface IPs or the system hostname in TLS certificate.
tlsdisableautofill=true
# Explicitly define any additional domain names for the certificate that will be created.
# tlsextradomain=raspibolt.local
# tlsextradomain=raspibolt.public.domainname.com


# Channel settings
bitcoin.basefee=1000
bitcoin.feerate=1
minchansize=100000
accept-keysend=true
accept-amp=true
protocol.wumbo-channels=true
coop-close-target-confs=24

# Watchtower
wtclient.active=true

# Performance
gc-canceled-invoices-on-startup=true
gc-canceled-invoices-on-the-fly=true
ignore-historical-gossip-filters=1
stagger-initial-reconnect=true

# Database
[bolt]
db.bolt.auto-compact=true
db.bolt.auto-compact-min-age=168h

[Bitcoin]
bitcoin.active=true
bitcoin.mainnet=true
bitcoin.node=bitcoind
```

Bitcoin
```
prune=550
server=1

# Network
listen=1
bind=127.0.0.1

# Connections
rpcauth=beno:<redacted rpc auth string>
zmqpubrawblock=tcp://127.0.0.1:28332
zmqpubrawtx=tcp://127.0.0.1:28333
whitelist=download@127.0.0.1
```

**Comment by bitromortac** [comment, marked as answer]
The recovery process is described here: https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md. A prerequisite to restore the funds is to restore the wallet. It seems like you entered a correct seed, but the passphrase is wrong. Try to think about whether the passphrase indeed belongs to the seed and check each character carefully, perhaps you made a mistake confusing any letters.

**Reply by tioneb22** [reply]
I've read through the document before attempting the recovery.

Do my error messages mean that my wallet password is wrong? It can't be that my seed passphrase is wrong because I didn't use an extra passphrase (25th word) to build my private key.

Also, I know my wallet password isn't wrong, because on my node's HDD I saved it in a password.txt file which I used to auto unlock LND when my server was rebooted.

Thanks

**Reply by guggero** [reply]
Did you use the auto unlock feature on your node before? Could it be that somehow there's a newline or space (or other un-printable) character in the password file?

**Reply by tioneb22** [reply]
I've always used the auto-unlock feature from the start. I still have access to my password.txt file (I mounted my nodes HDD in another VM) so I know for a fact I'm using the proper wallet password.

**Reply by guggero** [reply]
Though you are setting a **new** wallet password when calling `lncli create`. So is it possible there is already a `.lnd` data directory present on the system you are attempting recovery? Or is the password too short or contains weird characters? Can you try with a different, simple password? Since you said you didn't use a password for the seed, when restoring a node from the seed you don't need to use the wallet password (as that was just used to encrypt the file on disk in the old node).

**Reply by tioneb22** [reply]
Hmmm I see, did LND perhaps change what are valid characters for the wallet password? I used some underscores in the password... I'll try to make another simple password when I get home from work tonight. Will send update then... Thanks!

**Reply by tioneb22** [reply]
I tried entering a simpler password (only lower case character) but I still get the same error.

**Reply by tioneb22** [reply]
Would the wallet-unlock settings in LND be causing any issues? Keep in mind that I haven't actually created a wallet yet...
Also, my bitcoin node is a pruned, not sure if that makes a difference....

**Reply by bitromortac** [reply]
Just to clarify, you get `[lncli] rpc error: code = Unknown desc = invalid passphrase` after entering your wallet seed without a passphrase? If this is the case, it indicates that you entered a passphrase when the seed was created for the first time.

**Reply by guggero** [reply]
Try commenting out
```
wallet-unlock-password-file=/data/lnd/password.txt
wallet-unlock-allow-create=true
```

as well just to make sure that is not interfering. And make sure there is no `.lnd` directory on the machine you are attempting the restore.

**Comment by isamurad99** [comment]
interesting

**Comment by LunaLi777** [comment]
Thanks

**Comment by tioneb22** [comment]
FYI the selected answer isn't right. The recovery process never worked even if I had the correct passphrase (this is something I wrote down). The only way I recovered my funds was to migrate the VM to another server which had spare HDD space and spin it back up. I then closed my channels and sent my UTXOs to my cold wallet. I'll be honest, this experience scared me, because I had all the information to recover the funds, but it didn't work. Just thought I'd chime in a year later to explain what happened. Stay safe out there folks!
